src/app/public/src/modules/tutorial/tutorial-summary.component.scss

```diff
@@ -6,13 +6,13 @@
   text-align: center;
   margin: 0 0 20px;
 
-  @media (min-width: $stache-screen-sm) {
+  @media (min-width: $stache-screen-sm-min) {
     font-size: 19.5px;
   }
 
   &,
   p {
-    color: $stache-color-gray;
+    color: $stache-text-color-deemphasized;
```

Comment:
EDIT-   It Fails the AAA check for normal font.   passes the AA.  Not sure what standard we are aiming for here, but I figured AAA was the goal.